sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/SafeAddTests.java

```diff
@@ -0,0 +1,50 @@
+package org.dbsp.sqlCompiler.compiler.sql.simple;
+import org.dbsp.sqlCompiler.compiler.sql.SqlIoTest;
+import org.junit.Test;
+
+public class SafeAddTests extends SqlIoTest {
```

Comment:
unless some of these tests do not work you can probably put them into a single larger test with qs.
You should probably have one for integers and one for decimals.
You probably need to try various types of integers, which you can force using `CAST(value as TINYINT)`